lib/tokens.js

```diff
@@ -479,12 +492,13 @@ async function _getKek({hmac, version, salt}) {
 
 async function _createBatch({internalId, tokenVersion, tokenCount = 0}) {
   const {options: {batchIdSize, batchTokenCount}} = tokenVersion;
+  const _randomBytesAsync = promisify(crypto.randomBytes);
```

Comment:
please add a comment that this is done like this in support of stubbing in the test suite.